samples/boards/nrf52/mesh/onoff-app/src/main.c

```diff
@@ -0,0 +1,772 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This application is specific to the Nordic nRF52840-PDK board.
+ *
+ * It supports the 4 buttons and 4 LEDs as mesh clients and servers.
+ *
+ * Prior to provisioning, a button inverts the state of the
+ * corresponding LED.
+ *
+ * The unprovisioned beacon uses the device address set by Nordic
+ * in the FICR as its UUID and is presumed unique.
+ *
+ * Button and LED 1 are in the root node.
+ * The 3 remaining button/LED pairs are in element 1 through 3.
+ * Assuming the provisioner assigns 0x100 to the root node,
+ * the secondary elements will appear at 0x101, 0x102 and 0x103.
+ *
+ * It's anticipated that after provisioning, the button clients would
+ * be configured to publish and the LED servers to subscribe.
+ *
+ * If a LED server is provided with a publish address, it will
+ * also publish its status on a state change.
+ *
+ * Messages from a button to its corresponding LED are ignored as
+ * the LED's state has already been changed locally by the button client.
+ *
+ * The buttons are debounced at a nominal 250ms. That value can be
+ * changed as needed.
+ *
+ */
+
+/*
+ * Open Issues
+ *
+ * 1. Should the level skeleton remain or just be noted as still just
+ *    a skeleton?
+ *
+ *    OTOH: It would display level model messages sent by the SILabs
+ *    Android app. That may or may not have any educational value.
+ *
+ */
+
+#include <misc/printk.h>
+#include <misc/byteorder.h>
+#include <nrf.h>
+#include <device.h>
+#include <gpio.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/l2cap.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/mesh.h>
+
+#include <board.h>
+
+#define CID_INTEL 0x0002
+
+/* Model ID's */
+#define BT_MESH_MODEL_ID_GEN_ONOFF_SRV             0x1000
+#define BT_MESH_MODEL_ID_GEN_ONOFF_CLI             0x1001
```

Comment:
These are actually already defined in the public <bluetooth/mesh/access.h> header file which gets included when you include <bluetooth/mesh.h>

Reply:
Fixed